content/blog/2018/02/2018-02-22-cheetah.adoc

```diff
@@ -0,0 +1,134 @@
+---
+:layout: post
+:title: "Project Cheetah - Faster, Leaner Pipeline That Can Keep Up With Demand"
+:tags:
+- pipeline
+- performance
+- scalability
+- features
+:author: svanoort
+---
+
+:toc: 
+
+== The Problem To Solve
+Since it launched, Pipeline has had a bit of a Dr. Jekyll and Mr. Hyde performance problem.  In certain circumstances, Pipeline turns from a mild-mannered CI/CD assistant into a monster.  It will happily eat storage read/write capacity like popcorn with nary a care for the other concerns of our friendly butler.  When combined with additional factors, this results in real-world stability problems.  For example, combining slow storage with a spike in running Pipelines has brought down production Jenkins at more than one company.  Similarly users see issues if a busy master gets hit with an extra source of stress; past culprits have been heavy (ab)use of automated APIs, now-solved user lookup bugs, backup jobs, and plugins run amuck that load excessive numbers of builds.
```

Comment:
I'd suggest some more conditionality - , i.e., "Pipeline turns" -> "Pipeline can turn", "this results in..." -> "this can result in...". 
Also, comma after "Similarly" in "Similarly users", plz.